landing_page_backend/contacts/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from contacts.models import Contact, Project
+
+
+class OngoingProjectSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Project
+        fields = ('project_name', 'image', 'size', 'project_type')
+
+
+class PreviousProjectSerializer(serializers.ModelSerializer):
```

Comment:
Use init method to update already existing fields. See Two Scoops of Django 12.4 Pattern 4: Hacking Form Fields
(2 CBVs, 2 Forms, 1 Model)